src/server/api/controllers/channel-members.controller.js

```diff
@@ -42,6 +42,16 @@ const createChannelMember = async (body) => {
     successful: true,
   };
 };
+
+const checkForGeneralChannels = async (arrOfUsers) => {
+  const generalChannels = await knex('channel_members')
+    .select('fk_channel_id as channelId')
+    .whereIn('fk_user_id', arrOfUsers)
+    .groupBy('fk_channel_id')
+    .having(knex.raw('count(*)'), '=', arrOfUsers.length);
+  return generalChannels.map((channel) => channel.channelId);
+};
+
```

Comment:
This is what @wael85 did in a seperate PR. This is why communication is **so** important

Reply:
I know about that :) I tolk with Wael about that many times. But our api it is not the same.